app/services/autocomplete_search_service.rb

```diff
@@ -45,20 +45,12 @@ def find_suggestions # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
     #
     # rubocop:enable Style/AsciiComments
     #
-    results = response.body
-                      .force_encoding(Encoding::UTF_8)
-                      .split("\n")
-
-    if results.length > MAX_NUM_SUGGESTIONS
-      msg = <<~EO_MSG
-        Received #{results.count} autocomplete suggestions (#{results.count - MAX_NUM_SUGGESTIONS} more than expected):
-          #{results}
-      EO_MSG
-
-      @logger.info(msg)
-    end
-
-    results.take(MAX_NUM_SUGGESTIONS)
+    response
+      .body
+      .force_encoding(Encoding::UTF_8)
```

Comment:
Do we still need this? Had it been fixed on freelex?

Reply:
I haven't heard anything from Dave yet so I assume not. I'll ask again just in case.